src/main/java/no/api/freemarker/java8/time/DateTimeTools.java

```diff
@@ -113,4 +114,12 @@ public static ZoneId zoneIdLookup(List list, int index) throws TemplateModelExce
         return zoneId;
     }
 
+    private static Locale getLocale() {
+        if (Environment.getCurrentEnvironment() != null) {
+            return Environment.getCurrentEnvironment().getConfiguration().getLocale();
```

Comment:
Turns out we don't ned getConfiguration() in there since Environment extends Configurable and all getConfiguration() does is returning parent.